src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivitySourceTests.cs

```diff
@@ -125,6 +125,36 @@ public void TestActivityWithListenerActivityCreateAndAllDataRequested()
             }).Dispose();
         }
 
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void TestActivityTriggeringCallerMemberNameAttribute()
+        {
+            RemoteExecutor.Invoke(() => {
+                using (ActivitySource aSource = new ActivitySource("SourceActivityTriggeringCallerMemberNameAttribute"))
+                {
+                    using ActivityListener listener = new ActivityListener();
+                    listener.ShouldListenTo = (activitySource) => object.ReferenceEquals(aSource, activitySource);
+                    listener.Sample = (ref ActivityCreationOptions<ActivityContext> activityOptions) => ActivitySamplingResult.AllDataAndRecorded;
+
+                    ActivitySource.AddActivityListener(listener);
+
+                    string methodName = MethodBase.GetCurrentMethod().Name;
+
+                    using (Activity activity = aSource.StartActivity()) // passing default name should trigger CallerMemberName attribute.
```

Comment:
Do we need to test that things like tags, links, startTime, etc. are appropriately propagated?

Reply:
We already testing those with the other overloads. I can add some more validations for this overload.